### PR TITLE
Fix #782: Adding check for network connection in activities and fragments.

### LINKED ISSUE #782: Adding check for network connection in activities and fragments.
**Summary:**

In many fragments, while showing error, network connectivity is not checked which leads to a less suitable error message like "error loading in {Feature name}" whereas the message should reflect that there is no internet connection.

**Screenshots:**

screenshot_20180323-070210

### PR DESCRIPTION
**Pull request by jaindiv26**
Description: 
Adding check for network connection in activities and fragments.

Fixes #782 
Adding check for network connection in activities and fragments.

Screenshots: 
screenshot_1535994195

### CODE REVIEW

File: app/src/main/java/org/mifos/mobilebanking/utils/Network.java [hunk 1]
```diff
@@ -22,6 +23,30 @@ public static NetworkInfo getNetworkInfo(Context context) {
...
+     */
+
+    public static boolean isInternetOn(Context context) {
+        ConnectivityManager cm = (ConnectivityManager) context
+                .getSystemService(Context.CONNECTIVITY_SERVICE);
+        // test for connection
+        if (cm.getActiveNetworkInfo() != null
+                && cm.getActiveNetworkInfo().isAvailable()
+                && cm.getActiveNetworkInfo().isConnected()) {
+            Log.v("log", "Internet is working");
```
Comment (luckyman20) [inline app/src/main/java/org/mifos/mobilebanking/utils/Network.java:40]: Log commands should be removed

File: app/src/main/java/org/mifos/mobilebanking/utils/Network.java [hunk 2]
```diff
@@ -22,6 +23,30 @@ public static NetworkInfo getNetworkInfo(Context context) {
...
+            // txt_status.setText("Internet is working");
```
Comment (luckyman20) [inline app/src/main/java/org/mifos/mobilebanking/utils/Network.java:41]: Remove this comment also

**Comment by ivary43** [comment]
@jaindiv26  fix the travis CI issue.

**Comment by therajanmaurya** [comment]
First Implementation is totally wrong. You have added extra code that is already present in utils classes. Solution is to check internet connection from util class and show no network UI of sweet error.

**Pull request closed without merging**